### ISSUE TITLE
[* Duration] Extract duration units when preceded by ordinal numbers

### ISSUE THREAD

**Issue by LionbridgeCSII** [microsoft/Recognizers-Text#2417]
Duration units like "week", "month", "year"... should be recognized as duration entities when preceded by ordinal numbers.
For example: 
"the first week is dedicated to exercises"  --> "week" (duration)

**Comment by tellarin** [comment]
@LionbridgeCSII, can you take a stab at this and see if there may be any side-effects?

**Comment by LionbridgeCSII** [comment]
@tellarin, yes there are some interferences with DatePeriod and now some patterns are not correctly handled in certain languages. I am working on fixing these issues.

**Comment by tellarin** [comment]
Please raise a flag if this becomes too unwieldy.
